react-app/src/components/ProposalDetailScreen/ProposalHeader.tsx

```diff
@@ -31,10 +35,20 @@ const ProposalHeader: React.FC<{ proposal: Proposal }> = ({ proposal }) => {
     [status]
   );
 
+  const totalDepositString = useMemo(() => {
+    return (
+      convertBigNumberToLocalizedIntegerString(
+        convertMinimalTokenToToken(depositTotal)
```

Comment:
I think `convertMinimalTokenToToken` should be done in `useProposalQuery`. I would like to keep all minimal token <-> token conversion in api invoking level. We have had these conversion scattered around before and it appeared to be very confusing :sosad: